secant/Screens/Loading/LoadingScreen.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  LoadingScreen.swift
+//  secant-testnet
+//
+//  Created by Francisco Gindre on 9/2/21.
+//
+
+import SwiftUI
+
+protocol LoadingScreenRouter: AnyObject {
+    func proceedToHome()
+    func failWithError()
+    func proceedToWelcome()
+}
+
+struct LoadingScreen: View {
+    @State var router: LoadingScreenRouter?
+    
+    @StateObject var viewModel: LoadingScreenViewModel
```

Comment:
What do you think has higher precedence, State or StateObject?  Im inclined to say StateObject because it's meant to be the single source of truth for that object through all of it's ancestors if I understand correctly.  Which if true, would mean that @StateObject belongs on top of @State variables.  I think that's how the template is setup now too.